app/src/main/java/com/lec/spring/controller/RecipeController.java

```diff
@@ -0,0 +1,79 @@
+package com.lec.spring.controller;
+
+import com.lec.spring.entity.Recipe;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/recipe")
+public class RecipeController {
+
+
+    private static final Map<Long, Recipe> recipes = new HashMap<>();
+    private static Long id = 0L;
```

Comment:
```suggestion
    private Long id = 0L;
```

Sington이니까, 똑같이 동작하지 않을까요?